pages/summary.md

```diff
@@ -0,0 +1,38 @@
+# Summary
+
+[Introduction](Index.md)
```

Comment:
macOS is not case sensitive (by default), but our server likely is. We should either rename this `index.md` or rename the file `index.md -> Index.md`

Reply:
@mehtaishita  linux is case sensitive so SUMMARY.md -> summary.md in linux wont compile and i believe so for windows
Fix : let it be SUMMARY.md